Sound/Tidal/Pattern.hs

```diff
@@ -842,6 +842,7 @@ d1 $ sound (samples "xx(3,8)" (tom $ choose ["a", "e", "g", "c"]))
 plays a melody randomly choosing one of the four notes \"a\", \"e\", \"g\", \"c\".
 -}
 choose :: [a] -> Pattern a
+choose [] =  throw (ErrorCall "Empty list. Nothing to choose from.")
```

Comment:
Hi @vivid-synth, with `error`... the returned message is a bit more unclear with its additional compiler info. For live coding is that relevant? About erroring or returning `silence`, I have not chosen silence because the issue says to "Raise error (...)", I guess to give some info of what is happening.